### PR TITLE
[GRIFFIN-362] Add postgresql and oracle driver into dependencies.

### PR DESCRIPTION
**Pull request by lipzhu**
**What changes were proposed in this pull request?**
1. Add Oracle and postgresql JDBC driver into dependencies in measure module due user report 
https://issues.apache.org/jira/browse/GRIFFIN-362
2. Update postgresql jdbc driver version to the latest in service module. 

**Does this PR introduce any user-facing change?**
No.

How was this patch tested?
Unit Tests

### CODE REVIEW

**Comment by chitralverma** [comment]
@lipzhu  is there a valid JIRA ticket for this feature?
If not then please create one, link it to this PR and also add a valid description to the PR.

Most of the PRs follow this process.
Ref: https://github.com/apache/griffin/pull/593

File: measure/pom.xml [hunk 1]
```diff
@@ -53,6 +53,8 @@ under the License.
         <scalaj.version>2.3.0</scalaj.version>
         <mongo.version>2.1.0</mongo.version>
         <curator.version>2.10.0</curator.version>
+        <postgresql.version>9.4.1212.jre7</postgresql.version>
```
Comment (chitralverma) [inline measure/pom.xml:56]: This version is compiled with java7 and is from 2016. Please use a more recent and stable version.

**Review by chitralverma — APPROVED** [review]

File: measure/pom.xml (hunk 1 above)
Comment (lipzhu) [reply measure/pom.xml]: > @lipzhu is there a valid JIRA ticket for this feature? If not then please create one, link it to this PR and also add a valid description to the PR.
> 
> Most of the PRs follow this process. Ref: #593

@chitralverma Thanks for you review, this PR try to resolve https://issues.apache.org/jira/browse/GRIFFIN-362.

**Review by whhe — CHANGES_REQUESTED** [review]
There is a failed test case which is related to the added pgsql dependency.

https://github.com/apache/griffin/blob/ecb4d2b11223d52e7f47d11f5cf242a8a3fa429c/measure/src/test/scala/org/apache/griffin/measure/datasource/connector/batch/JDBCBasedDataConnectorTest.scala#L135-L146

**Review by whhe — APPROVED** [review]

**Review by chitralverma — APPROVED** [review]

**Comment by chitralverma** [comment]
LGTM, merging.

**Pull request closed without merging**